frontend/src/pages/RecipePage.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components/macro'
+import {useState, useEffect} from "react";
+import {useParams} from "react-router-dom";
+import {getRecipe} from "../service/recipeApiService";
+import RecipeHeader from "../components/RecipeHeader";
+import RecipeCoronaDetails from "../components/RecipeCoronaDetails";
+import RecipePreparation from "../components/RecipePreparation";
+
+
+export default function RecipePage() {
+    const {recipeId} = useParams()
+    const [recipe, setRecipe] = useState()
+
+    useEffect(() => {
+        getRecipe(recipeId).then(setRecipe)
+    }, [])
```

Comment:
```suggestion
    }, [recipeId])
```
You should  at this as dependency. otherwise you dont load another recipe when the parameter changes